routes/api/branches.js

```diff
@@ -1,8 +1,67 @@
 const express = require("express");
 const router = express.Router();
 
```

Comment:
Also, rename all those files to be in singular - `branch.js` instead of `branches.js`, etc for all of them. It makes more sense and it will be consistent with the model name which is important

Reply:
Do you want it to also be changed in the app.js file to be singular routes?

Reply:
Yeah, I think it would be more clear &  searchable if we keep it singular consistently